src/parser.cc

```diff
@@ -199,14 +199,14 @@ std::vector<std::pair<std::string, bool>> tokenize_line(const std::string& line)
 
 std::vector<std::pair<std::string, bool>> split_tokens(std::vector<std::pair<std::string, bool>> tokens) {
     std::vector<std::pair<std::string, bool>> result;
-    for (const auto& token : tokens) {
+    for (auto& token: tokens) {
         if (token.second) { // is quoted?
             result.push_back(std::move(token));
             continue;
         }
 
-        const std::string_view token_string = token.first;
-        size_t last_operator = 0;
+        const std::string_view token_string{ token.first };
```

Comment:
why this everywhere actually?

Reply:
I mena why to initialize things with the curly brackets {bla} instead of = bla?

Reply:
I think it is some C++ idiom, that is advised to use. I myself don't like it.

Reply:
It is to prevent narrowing of the types (https://stackoverflow.com/a/27755143), but naturally, C++ community has to make sure it is ugly as pug's arse.

Reply:
Exactly as you say. It is a good idea, only horribly implemented as far as visualization goes. But it helps indeed and warns you about potential implicit conversion bugs. In modern C++, it should always be preferred over assignment operator.